### PR TITLE
add markup and style for hello microverse

### PR DESCRIPTION
**Pull request by Sammy3000**
## In this section I did the following:
 - Add markup and style for hello microverse.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/
```
Comment (anagudelogu) [inline .gitignore:2]: - [ ] You did a great job working on this project! 💯 However, you missed an important requirement:

image

Please consider creating a file named `test.md` and then adding it to your `.gitignore` file! 🤓 🛠️

**Review by anagudelogu — CHANGES_REQUESTED** [review]
# STATUS:  CHANGES REQUIRED  ♻️ 🚧 

Hi @Sammy3000, 
I'm Andres.


While you have done a great job working on this milestone 💻 , there are some issues that you still need to work on to go to the next project. ♻️ 

You are almost there, keep up the good work! 🚀 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Descriptive PR title, summary and commit messages. 💬 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -149,26 +102,14 @@ Example commands:
...
-<!--
-Example command:
-
-```sh
-  rails server
-```
---->
+- Feel free to use this project.
 
 ### Run tests
```
Comment (DuaneDave) [inline README.md:115]: [OPTIONAL] In order to have a solid documentation, it is advisable you add all command that is required in testing this project

for example

`
npx hintrc .
`

**Review by DuaneDave — APPROVED** [review]
Hi @Sammy3000 👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Professional readme 💯 
✔️ Linters are passing 💯 

## Optional suggestions
- A pull request should be explanatory such that it highlights most of the activities you have carried out in that PR so I suggest that you add more information in your PR summary in future projects
- In order to have a solid documentation, it is advisable you add all command that is required in testing this project
- Also, consider starting every commit messages with a capital letter to make them more professional 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**